aptos-move/framework/aptos-token/doc/token.md

```diff
@@ -1141,7 +1141,7 @@ The field is not mutable
 Withdraw capability doesn't have sufficient amount
 
 
-<pre><code><b>const</b> <a href="token.md#0x3_token_EINSUFFICIENT_WITHDRAW_CAPABILITY_AMOUNT">EINSUFFICIENT_WITHDRAW_CAPABILITY_AMOUNT</a>: u64 = 37;
+<pre><code><b>const</b> <a href="token.md#0x3_token_EINSUFFICIENT_WITHDRAW_CAPABILITY_AMOUNT">EINSUFFICIENT_WITHDRAW_CAPABILITY_AMOUNT</a>: u64 = 38;
```

Comment:
is this supposed to be here?

Reply:
probably not, I'll merge and get rid of it. Why does it keep appearing in the diffs? @zekun000 @areshand

Reply:
I think running the linter does it?

Reply:
and / or some tests